cl/api/templates/rest-docs-vlatest.html

```diff
@@ -1036,107 +1057,51 @@ <h3 id="visualization-endpoint">{% url "scotusmap-list" version="v3" %}
     <p>Similar commands can be used to update other non-readonly fields if desired.</p>
     <p>Visualizations can be marked as deleted by flipping the <code>deleted</code> field to <code>True</code>, or can be deleted forthright via a <code>DELETE</code> request.</p>
 
-    <h2>Footnotes</h2>
-    <p id="fn1"><sup>1</sup> The <code>search</code> endpoint is handled by our search engine, not our database. Filters and ordering are thus applied using the same <code>GET</code> parameters as are created when using the front end of the website. Our recommendation is to hone your query there, then move the parameters to the API. See more details in <a href="#search-endpoint">the search section of this document</a>.
+    <h2 id="available-jurisdictions"><span>Available Jurisdictions</span></h2>
+    <p>We currently have <strong>{{ court_count }}</strong> jurisdictions that can be accessed with our APIs. Details about the jurisdictions that are available, including all abbreviations, can <a href="{% url "court_index" %}">be found here</a>.
     </p>
-    <p><a href="#fn-ref-1">Back to reference 1 &#8617;</a></p>
-    <p><a href="#fn-ref-2">Back to reference 2 &#8617;</a></p>
-  </div>
-
 
 
-  <div class="col-sm-5">
-    <div class="well">
-      <h3 class="top">Your Authorization Token</h3>
-      {% if user.is_authenticated %}
-        <p>Your permanent token for use with this API is:</p>
-        <h4><code>{{ user.auth_token }}</code></h4>
-        <p>Do not share your API token. Keep it private like a password.
-        </p>
-        <p class="text-right"><a class="btn btn-default" href="{% url "view_api" %}#usage">See Usage</a></p>
-      {% else %}
-        <p><a href="{% url "sign-in" %}?next={{ request.path }}"
-                class="btn btn-primary btn-lg">Sign In To See Your Token</a>
-        </p>
-      {% endif %}
-    </div>
+    <h2 id="powered-by"><span>Powered By</span></h2>
+    <p> If you are interested, we have a variety of <a href="https://free.law/press-assets/">"Powered By" logos</a> available for your website or application. We appreciate a reference, and it supports our work, but it's not currently required.
+    </p>
+    <a href="https://free.law/press-assets/">
+      <img src="{% static "png/logo-long-w350.png" %}" width="350" height="81">
+    </a>
 
-    {% include "includes/toc_sidebar.html" %}
+    <h2 id="citegeist-scores"><span>CiteGeist Scores</span></h2>
+    <p>If you are interested in the CiteGeist score for each opinion, it is now available via our <a href="{% url "bulk_data_index" %}#citegeist">bulk data API</a>.
+    </p>
+    <p>This data is not available via the REST API because a single new citation between cases causes a ripple effect through every value in the database.
+    </p>
```

Comment:
Let's remove this too. We don't share this file anymore (it was too complex).

Reply:
Done.